pkg/pillar/cmd/zedrouter/probe.go

```diff
@@ -104,7 +102,7 @@ func niUpdateNIprobing(ctx *zedrouterContext, status *types.NetworkInstanceStatu
 			niProbingUpdatePort(ctx, *devPort, status)
 		}
 	}
-	checkNIprobeUplink(ctx, status)
+	checkNIprobeUplink(ctx, status, "")
```

Comment:
Is the assumption there that network instance port never changes?

Reply:
Basically whatever the NI status changes, has nothing to do with if we are going to remove an existing port. Only device Network status changes that.

Reply:
ok